bson/bsoncodec/pointer_codec.go

```diff
@@ -14,11 +14,6 @@ import (
 	"go.mongodb.org/mongo-driver/bson/bsontype"
 )
 
-var defaultPointerCodec = &PointerCodec{
```

Comment:
This was unused and shouldn't exist in non-test files anyway because it's not safe to share a `PointerCodec` instance across registries, so I removed it.